platform-hub-web/src/app/projects/projects-detail.html

```diff
@@ -334,8 +334,100 @@ <h4>{{m.user.email}}</h4>
+                        when="{'0': '0 robots',
+                        '1': '1 robot',
+                        'other': '{} robots'}">
+                      </ng-pluralize>
+                      <span class="separator"></span>
+                      <ng-pluralize
+                        count="d.access.users.length"
+                        when="{'0': '0 users',
+                        '1': '1 user',
+                        'other': '{} users'}">
+                      </ng-pluralize>
+                    </div>
+                    <md-expansion-panel-icon></md-expansion-panel-icon>
+                  </md-expansion-panel-header>
+                  <md-expansion-panel-content>
+                    <h3 class="md-subhead">Robot accounts</h3>
+                    <p
+                      ng-if="!d.access.robots.length"
+                      class="md-body-1 none-text">
+                      None
+                    </p>
+                    <md-list ng-if="d.access.robots.length" flex>
+                      <md-list-item class="md-2-line" ng-repeat="r in d.access.robots">
+                        <div class="md-list-item-text" flex>
+                          <h3>{{r.username}}</h3>
+                          <p>
+                            Status: {{r.status}}
+                          </p>
+                        </div>
+                        <md-divider ng-if="!$last"></md-divider>
+                      </md-list-item>
+                    </md-list>
+
+                    <br />
+
+                    <h3 class="md-subhead">Project team members</h3>
+                    <p
+                      ng-if="!d.access.users.length"
+                      class="md-body-1 none-text">
+                      None
+                    </p>
+                    <md-list ng-if="d.access.users.length" flex>
+                      <md-list-item class="md-2-line" ng-repeat="u in d.access.users">
+                        <div class="md-list-item-text" flex>
+                          <h3>{{u.username}}</h3>
+                          <p>
+                            <span>Write access? {{u.writable}}</span>
+                            <span class="separator">|</span>
+                            <span>Status: {{u.status}}</span>
+                          </p>
+                        </div>
+                        <md-divider ng-if="!$last"></md-divider>
+                      </md-list-item>
+                    </md-list>
+                  </md-expansion-panel-content>
+                </md-expansion-panel-expanded>
+              </md-expansion-panel>
             </md-card-content>
             <md-card-actions layout="row" layout-align="start center">
+              <md-button
+                ng-disabled="d.status != 'active'"
+                aria-label="Manage the access for this Docker repo"
+                ng-click="$ctrl.dockerRepoManageAccess(d, $event)">
+                Manage Access
```

Comment:
Access management doesn't appear to be scoped to project admins, just confirming this is intentional right? So the plan initially is that all project members can create, delete and control access for docker repos.

Reply:
Yep, this is intentional and a key design decision of this new piece of work (i.e. that project team members have more autonomy over management of resources).